ansible/roles/products/common/demo-icons/tasks/main.yml

```diff
@@ -133,28 +144,30 @@
   - name: Create Open Left Hand desktop icon
     template:
       src: ../resources/templates/desktop-icons/standard-icon.j2
-      dest: "{{ user_folder }}/Desktop/Open_Left_Hand.desktop"
+      dest: "{{ demo_icon_folder }}//Open Left Hand.desktop"
       mode: '755'
     vars:
       desktop_shortcut_name: Open Left Hand
       comment: "This opens the left hand"
       folder: "{{ shadow_hand_launcher_folder }}"
       shell_script_file_name: open_left_hand.sh
       icon_file_name: open-hand-icon-left.png
+      start_terminal: "true"
 
   - name: Make Open Left Hand desktop icon Trusted
-    shell: gio set "{{ user_folder }}/Desktop/Open_Left_Hand.desktop" "metadata::trusted" yes
+    shell: gio set "{{ demo_icon_folder }}//Open Left Hand.desktop" "metadata::trusted" yes
     when:
       - ansible_distribution|string == 'Ubuntu'
       - ansible_distribution_release|string == 'bionic'
 
-  when: bimanual|bool
+  when: bimanual|bool or hand_id=="lh"
```

Comment:
this should have been a new pr. Its also in another user story

Reply:
ok